src/pages/viewer/pane/ActionPane.svelte

```diff
@@ -8,7 +8,7 @@
   import { cancelActions } from "@/viewer/document.js";
 
   // SVG assets
-  import closeSvg from "@/assets/close.svg";
+  import { XCircle24 } from "svelte-octicons";
```

Comment:
Nice move here! In a separate PR, it'd be good to see what other in-house icons we can replace with the icon library. I can create an issue defining that task?

Reply:
There is one: #266 

I also added a couple scripts in the `utilities` folder to find where we're using SVGs.

Reply:
Oh, and I made this for reference: https://svelte.dev/repl/3fdb0f8f5ddd4950afbd53f995f9e49c?version=4.2.0